File: Feitos em Python 3/2518.py
import math
while True:
    try:
        degraus=int(input())
        vet=input().split()
        h=int(vet[0])
        c=int(vet[1])
        l=int(vet[2])
        compRampa=math.sqrt(pow(c,2)+pow(h,2))
        compRampa*=degraus
        area = compRampa*l
        print("%.4f"%(area/10000))
    except EOFError:
        break

File: Feitos em C Sharp/2518.cs
using System;
using System.Globalization;
namespace ConsoleApp26 {
    class Program {
        static void Main(string[] args) {
            while (true) {
                string input = Console.ReadLine();
                if (string.IsNullOrEmpty(input)) {
                    break;
                }
                int degraus = Convert.ToInt32(input);
                string[] vet = Console.ReadLine().Split();
                int h = Convert.ToInt32(vet[0]);
                int c = Convert.ToInt32(vet[1]);
                int l = Convert.ToInt32(vet[2]);
                double compRampa = Math.Sqrt(Math.Pow(c, 2) + Math.Pow(h, 2));
                compRampa *= degraus;
                double area = compRampa * l;
                Console.WriteLine((area/10000).ToString("F4",CultureInfo.InvariantCulture));
            }
        }
    }
}